docs/api.md

```diff
@@ -0,0 +1,233 @@
+# electron-packager API
+
+Short example:
+
+```javascript
+var packager = require('electron-packager')
+packager(options, function done_callback (err, appPath) { /* … */ })
```

Comment:
It's still error-object-first, it's just `(err, appPaths)` now rather than singular `appPath`.